src/styles.css

```
@@ -477,6 +477,44 @@ footer a:hover{text-decoration:underline}
+
+.sticky-controls{
+  position: sticky;
+  top: 70px;
+  z-index: 1000;
+
+  background: white;
+
+  padding: 12px 0;
+  width:100%;
+
+  box-shadow:0 2px 8px rgba(0,0,0,0,05);
```

Stylelint (17.11.1)

[error] Expected "rgba" to be "rgb" (color-function-alias-notation)

(color-function-alias-notation)